modules/mod_ginger_spa/templates/public/test.tpl

```diff
@@ -0,0 +1,2 @@
+{% debug %}
+{% print q.v %}
```

Comment:
Maybe this should be less dangerous (-looking) default content, without `debug`? I Imagine you'll easily forget it is there - wait..maybe if we change the default dispatch rule to explicitly reference test.tpl, you'd be bound to remove or fix it in an actual project.

Reply:
The debug just echos what is passed, and afaik all data is escaped.